### DISCUSSION TITLE
Docker daemon restart taking 10 Minutes

### DISCUSSION THREAD

**Discussion by aneesinaec** [moby/moby#47665] [General]
Between the following log lines, the docker daemon is spending around 10 minutes.

https://github.com/docker/engine/blob/8955d8da8951695a98eb7e15bead19d402c6eb27/daemon/graphdriver/driver.go#L205
https://github.com/docker/engine/blob/8955d8da8951695a98eb7e15bead19d402c6eb27/daemon/graphdriver/overlay2/overlay.go#L228

What might be causing the 10 minutes  delay?

**Comment by thaJeztah** [comment]
Those links are pointing to a legacy / archived codebase; what version of docker are you running? Can you post `docker version` and `docker info` output?
